vorpal/Base/custom_selenium_driver.py

```diff
@@ -0,0 +1,232 @@
+"""
+Package: Base
+WCustomSeleniumDriver class implementation.
+Methods for adding the new functionality to selenium methods.
+"""
+from selenium.common.exceptions import NoSuchElementException, ElementNotSelectableException, ElementNotVisibleException
+from selenium.webdriver.support import expected_conditions as ec
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.remote.webelement import WebElement
+from Base.utilities import Utilities
```

Comment:
Does `Base.utilities` exist in this project yet? That's a custom class/module you made, right?

Reply:
I'm not really sure what I'm looking at here. This looks like a PR for a bunch of helper methods. Is this something that's absolutely needed in a foundation for us to start our own test scripts?

The title is misleading, `Custom Selenium Driver` if this is just a PR for helper methods that might be useful in screen.

Using github to checkout and check-in code is a new concept and process for me, so please forgive my ignorant questions.